isamples_metadata/GEOMETransformer.py

```diff
@@ -500,8 +500,15 @@ def authorized_by(self) -> typing.List[str]:
         return []
 
     def complies_with(self) -> typing.List[str]:
-        # Don't have this information
-        return []
+        local_contexts_id = self.local_contexts_id()
+        if local_contexts_id is not None:
+            return [f"localcontexts:https://localcontextshub.org/projects/{local_contexts_id}"]
```

Comment:
Let's make this in the form common for identifiers (`scheme:prefix/value`), so:
`f"localcontexts:projects/{local_contexts_id}"`

Reply:
https://github.com/isamplesorg/isamples_inabox/pull/390/commits/5f4156e91f9e62dee46973a8d388d5c223bfd7d9